controllers/signout.js

```diff
@@ -0,0 +1,14 @@
+const signout = (req, res) => {
+  const cookies = Object.keys(req.cookies);
+  cookies.forEach((cookie) => {
+    res.clearCookie(cookie);
+  });
+  return res.json({
+    message: "Cookies deleted Succesfully",
```

Comment:
Do we need to be so verbose? Since the action is to logout we should provide a message related to that.
Something like `Signout successful!`

Reply:
Fixing it immediately.